src/huggingface_hub/inference/_client.py

```diff
@@ -2401,9 +2407,10 @@ def zero_shot_classification(
             text (`str`):
                 The input text to classify.
             labels (`List[str]`):
-                List of string possible labels. There must be at least 2 labels.
+                List of strings. Each string is the verbalization of a possible label for the input text.
             multi_label (`bool`):
```

Comment:
Can you add a docstring for `hypothesis_template` (with an example) between the docstrings for `multi_labels` and `model`?